app/graphql/mutations/profile/edit.rb

```diff
@@ -6,7 +6,7 @@ module Profile
     class Edit < BaseMutation
       graphql_name 'UpdateProfile'
 
-      POLICY_ATTRIBUTES = %i[name description
+      POLICY_ATTRIBUTES = %i[description
                              compliance_threshold business_objective_id].freeze
```

Comment:
The graphql schema is autogenerated and must be re-generated pretty much whenever we change anything within `app/graphql`: `rake dump_schema`

Reply:
Working on these updates, thanks for the review!